numpydoc/cli.py

```diff
@@ -100,8 +101,6 @@ def _parse_config(s):
     )
     lint_parser.add_argument(
         "--ignore",
-        type=str,
-        nargs="*",
         help=(
```

Comment:
I think we should indicate how this can be used in the help now that there are multiple options.

Reply:
@stefanv WDYT?